Model/Config/Source/TokenizedPaymentMethods.php

```diff
@@ -13,14 +13,15 @@ class TokenizedPaymentMethods implements OptionSourceInterface
     public function toOptionArray()
     {
         return [
+            /** TODO: These PMs can be enabled for recurring purposes once tested */
             /*[
                 'value' => PaymentMethods\ApplePayPaymentMethod::TX_VARIANT,
                 'label' => PaymentMethods\ApplePayPaymentMethod::NAME
-            ],*/
+            ],
             [
                 'value' => PaymentMethods\AmazonPayPaymentMethod::TX_VARIANT,
                 'label' => PaymentMethods\AmazonPayPaymentMethod::NAME
-            ],
+            ],*/
```

Comment:
Is this commented out code could be removed ?

Reply:
Nope, this code will be relevant once we enable tokenization on more payment methods.